packages/forms-web-app/src/views/layouts/default.njk

```diff
@@ -12,6 +14,15 @@
 	{{ pageTitle }}
 {% endblock %}
 
+{# Note this block can be removed as well as the relevent images in /public/images when updating to v5 of the govuk-frontend node module #}
```

Comment:
Just a suggestion but might be nice to put this chunk into an includes named 'head-icons.njk'.

Reply:
To be honest, I think it's more obvious here with the comment that it needs to be removed when the upgrade to govuk-frontend V5 is implemented.  The additional file would need to be deleted.  I will implement as you have described if you think it still makes sense for it to be in a separate file.